config.rb.in

```diff
@@ -0,0 +1,63 @@
+module BuildingSync
+
+  #ENV['GEM_HOME'] = 'C:/bricr/gems'
+  #ENV['GEM_PATH'] = 'C:/bricr/gems'
+  
+  # location of openstudio CLI
+  if /mswin/.match(RUBY_PLATFORM) || /mingw/.match(RUBY_PLATFORM)
+    OPENSTUDIO_EXE = 'C:/openstudio-2.6.0/bin/openstudio.exe'
+  else
+    OPENSTUDIO_EXE = '/usr/local/bin/openstudio'
+  end
+
+  # one or more measure paths
+  OPENSTUDIO_MEASURES = [
```

Comment:
we'll want to make these more configurable. Right now if these change the user will need to fork the gem or overload Module variables...

Reply:
This stuff is all covered by the extension gem.  Paths to measures in this gem are defined in the Extension class.  Paths from other extension gems are automatically discovered.

Reply:
removed the local constant and using the extension gem now. 
@nllong anything else we can to make these more con configurable? A user configuration file, or some additional parameters to the main entry functions?